src/main/resources/com/google/api/codegen/ruby/Rakefile.snip

```diff
@@ -4,11 +4,15 @@
   {@header(metadata.fileHeader)}
 
   {@unitTestsTask(metadata)}
-  @if smokeTestsTask(metadata)
+  @if metadata.hasSmokeTests
 
     {@smokeTestsTask(metadata)}
   @end
 
+  {@acceptanceTestTask(metadata)}
```

Comment:
This is needed because the metapackages Rakefile calls the `:acceptance` task on all the gems in the metapackage.

https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/Rakefile#L92